packages/studio-base/src/components/Preferences.tsx

```diff
@@ -188,6 +189,37 @@ function TimeFormat(): React.ReactElement {
   );
 }
 
+function LaunchDefault(): React.ReactElement {
+  const [preference = "unknown", setPreference] = useAppConfigurationValue<string | undefined>(
+    AppSetting.LAUNCH_PREFERENCE,
+  );
+
+  const entries: Array<{ key: string; text: string }> = [
+    { key: "unknown", text: "Ask Me" },
+    { key: "web", text: "Web" },
+    { key: "desktop", text: "Desktop" },
```

Comment:
just some ideas, feel free to take some or none of these suggestoins
```suggestion
    { key: "unknown", text: "Ask each time" },
    { key: "web", text: "Web app (studio.foxglove.dev)" },
    { key: "desktop", text: "Desktop app" },
```

Reply:
Good suggestions. I'll use these.